network-api/networkapi/buyersguide/templates/fragments/chapter_table_of_contents.html

```diff
@@ -16,7 +17,15 @@
             <div class="row publication-row pt-3 pb-3 d-flex align-items-center">
               <div class="col-sm-1 d-md-block d-none">&nbsp;</div>
               <div class="col-auto">
-                <div class="publication-chapter-number publication-chapter-number--small"></div>
+                <div class="publication-chapter-number publication-chapter-number--small"
+                  {% with toc_image=child.specific.toc_thumbnail_image %}
+                  {% if toc_image %}
+                  {% image toc_image fill-128x128 as thumbnail %}
```

Comment:
Help text suggests using a `320x320` image but the images in the templates are using `128x128`. It'll shrink down which might be what you want. Just thought I'd flag that here in case this was supposed to be `320x320`

Reply:
Nice flag.

Yea this is intentional. It seems like a publication page can serve two type of ToC - one with bigger thumbnail (chapter ToC) and one with smaller thumbnail (publication ToC). Since we can't distinguish them during page creation process, I set the help text using the larger thumbnail dimension so the image could render well for both cases.

![image](https://user-images.githubusercontent.com/2896608/105245058-37eb8980-5b26-11eb-8abf-bfe400000ba0.png)

and 

![image](https://user-images.githubusercontent.com/2896608/105245295-80a34280-5b26-11eb-8b9c-677665e39125.png)